src/main/groovy/edu/oregonstate/mist/api/PrettyPrintResponseFilter.groovy

```diff
@@ -0,0 +1,41 @@
+package edu.oregonstate.mist.api
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.databind.ObjectWriter
+import com.fasterxml.jackson.jaxrs.cfg.EndpointConfigBase
+import com.fasterxml.jackson.jaxrs.cfg.ObjectWriterInjector
+import com.fasterxml.jackson.jaxrs.cfg.ObjectWriterModifier
+import javax.ws.rs.container.ContainerRequestContext
+import javax.ws.rs.container.ContainerResponseContext
+import javax.ws.rs.container.ContainerResponseFilter
+import javax.ws.rs.core.MultivaluedMap
+
+@groovy.transform.TypeChecked
+public class PrettyPrintResponseFilter implements ContainerResponseFilter {
+
+    @Override
+    public void filter(
+            ContainerRequestContext requestContext,
+            ContainerResponseContext responseContext) throws IOException {
+
+        MultivaluedMap<String, String> queryParams =
+            requestContext.getUriInfo().getQueryParameters()
+
+        if (queryParams.getFirst('pretty') == 'true') {
```

Comment:
Ignoring case in a search parameter makes sense. Pretty printing is a developer feature—developers are used to details like case mattering. Who would we really be serving by making true case insensitive? I don't think remembering that "true" is lowercase is any harder than remembering that correct spelling is "true", not "yes" or "1" or "t". How far should we go? What about the other parts of the url? You wouldn't expect `/API/V1/Locations` to work, would you? Or `/api/v1/locations?PRETTY=true`? Your argument applies just as well there.